Sprint-3/implement/test/get-card-value.test.js

```diff
@@ -0,0 +1,4 @@
+const getCardValue = require("../get-card-value");
```

Comment:
The function is not designed to take two separate parameters, so the test is inconsistent with the current implementation.

Reply:
change it to one parameter